src/js/helpers/qajaxWrapper.js

```diff
@@ -0,0 +1,76 @@
+var qajax = require("qajax");
+
+var qajaxWrapper = function (options) {
+  var response = {
+    status: null,
+    body: null
+  };
+  // Translate Oboe options to qajax equivalent
+  var oboeDictionary = {
```

Comment:
Should we avoid building references to Oboe in directly, in case we want to remove it at some point?